src/psyclone/psyGen.py

```diff
@@ -3977,7 +4048,23 @@ def rename_and_write(self):
         fdesc = None
         while not fdesc:
             name_idx += 1
-            new_suffix = "_{0}".format(name_idx)
+            new_suffix = ""
+
+            # On the GOcean OpenCL work we need to differentiate between the
```

Comment:
"...between kernels generated from the same module file..."

Reply:
Done